src/app/components/user/user-profile/user-followers/user-followers.component.ts

```diff
@@ -7,9 +15,19 @@ import { Component, OnInit } from '@angular/core';
 })
 export class UserFollowersComponent implements OnInit {
 
-  constructor() { }
+  private subscription: Subscription;
+  private userIndex: string;
+  private followersList$: Observable<Array<UserProfile>>
```

Comment:
@zeus999 this should be done in parent component `Smart/Container Component`, as a result of this observable fetching we have to use `?` a lot in the presentation component which IMO is this one. @pkrawat1 @kartikjagdale Thoughts ?